src/modules/task.js

```diff
@@ -0,0 +1,76 @@
+import LocalStorage from './local_storage.js';
+/* eslint-env node */
+import Status from './status.js';
+
+const mainListCard = document.querySelector('.main-list-card');
+
+// create object from  localStorage
+const localS = new LocalStorage();
+
+// create status object from Status
+const status = new Status();
+
+class Task {
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
- [ ] Please remove the **eslint disable** statement. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  You just need to export it as the default code. :+1: